### PR TITLE
Document more necessary CaaSP settings (SCRD-8793)

### PR DESCRIPTION
**Pull request by cadenzajon**

### CODE REVIEW

File: xml/install_caasp_heat_templates.xml [hunk 1]
```diff
@@ -584,18 +598,26 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
        <screen>&prompt.ardana;zypper se --detail velum-image
 i | sles12-velum-image | package    | 3.1.7-3.27.3  | x86_64 | update_caasp</screen>
       </step>
      </substeps>
     </step>
     <step>
+     <para>
+      When preparing to bootstrap using the &caasp; web interface, set the
+      <literal>Internal Dashboard Location</literal> to the private network
+      IP address (i.e. in the the 172.x.x.x subnet) of the &cassp; admin node
```
Comment (csymons-suse) [inline xml/install_caasp_heat_templates.xml]: entity should be &caasp; not &cassp;

File: xml/install_caasp_heat_templates.xml [hunk 2]
```diff
@@ -528,6 +528,14 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
       </para>
      </listitem>
     </varlistentry>
+    <varlistentry>
+     <term>dns_nameserver</term>
+     <listitem>
+      <para>
+       Set this to the IP address of a DNS nameserver which will be reachable by the &caasp; cluster.
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]:
```suggestion
Set this to the IP address of a DNS nameserver so it is reachable by the &caasp; cluster.
```

Comment (asettle) [reply xml/install_caasp_heat_templates.xml]: Open to suggestions here, but removing future tense regardless :)

File: xml/install_caasp_heat_templates.xml [hunk 3]
```diff
@@ -571,6 +579,12 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
         Log in to each node and add the update repository.
        </para>
        <screen>&prompt.ardana;sudo zypper ar http://nu.novell.com/SUSE/Updates/SUSE-CAASP/3.0/x86_64/update/ caasp_update</screen>
+       <para>
+        Note the importance of having set a valid DNS server in your environment XML
+        file, as <literal>nu.novell.com</literal> must be reachable in order to
+        proceed with the update. (You may attempt <command>ping nu.novell.com</command>
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: Feels clunky, suggested rewrite:

_<literal>nu.novell.com</literal> must be reachable in order to proceed with the update, ensure you set a valid DNS server in your environment XML file._

File: xml/install_caasp_heat_templates.xml [hunk 4]
```diff
@@ -584,18 +598,26 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
 i | sles12-velum-image | package    | 3.1.7-3.27.3  | x86_64 | update_caasp</screen>
       </step>
      </substeps>
     </step>
     <step>
+     <para>
+      When preparing to bootstrap using the &caasp; web interface, set the
+      <literal>Internal Dashboard Location</literal> to the private network
+      IP address (i.e. in the the 172.x.x.x subnet) of the &caasp; admin node
+      created during the <filename>heat-caasp-deploy.yml</filename> playbook,
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: playbook. *

File: xml/install_caasp_heat_templates.xml [hunk 5]
```diff
@@ -571,6 +584,12 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
         Log in to each node and add the update repository.
        </para>
        <screen>&prompt.ardana;sudo zypper ar http://nu.novell.com/SUSE/Updates/SUSE-CAASP/3.0/x86_64/update/ caasp_update</screen>
+       <para>
+        <literal>nu.novell.com</literal> must be reachable in order to proceed
+        with the update, which requires a valid DNS server having been configured.
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: Sorry, the "which" here  doesn't make sense. My suggestion is to use "ensure a valid DNS server has been configured." or something similar :)

File: xml/install_caasp_heat_templates.xml [hunk 6]
```diff
@@ -571,6 +584,12 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
...
+        (You may attempt <command>ping nu.novell.com</command> at this time to
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: I would suggest "You can attempt..."

File: xml/install_caasp_heat_templates.xml [hunk 7]
```diff
@@ -563,39 +576,62 @@ git clone https://github.com/SUSE/caasp-openstack-heat-templates
       admin node bootstrapping steps.
      </para>
      <para>
-      Apply the &caasp; 3.0 Maintenance Update with the following steps:
+      Apply the &caasp; 3.0 Maintenance Update with the following steps,
+      run on each node in the cluster. This can be done one at a time,
```
Comment (asettle) [inline xml/install_caasp_heat_templates.xml]: I'm sorry, I must have missed this the first time "steps, run on each node in the cluster." -> please remove the comma!

**Pull request merged**